File: app/src/main/java/com/example/myapplication/model/Evento.java
package com.example.myapplication.model;

public class Evento {

    private int id;
    private String descricao;
    private int idData;
    private String Data;

    public Evento(String descricao, int idData) {
        this.descricao = descricao;
        this.idData = idData;
    }
    public Evento(){};

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public int getIdData() {
        return idData;
    }

    public void setIdData(int idData) {
        this.idData = idData;
    }

    public String getData() {
        return Data;
    }

    public void setData(String data) {
        Data = data;
    }
}

File: SERVIDOR/Model/Evento.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Evento
 *
 * @author luan
 */
class Evento {
    private $id;
    private $descricao;
    private $idData;
    private $idRegistro;
    
    function __construct($descricao, $idData, $idRegistro) {
        $this->descricao = $descricao;
        $this->idData = $idData;
         $this->idRegistro = $idRegistro;
    }
    function getIdRegistro() {
        return $this->idRegistro;
    }

    function setIdRegistro($idRegistro) {
        $this->idRegistro = $idRegistro;
    }

        
    function getId() {
        return $this->id;
    }

    function getDescricao() {
        return $this->descricao;
    }

    function getIdData() {
        return $this->idData;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setDescricao($descricao) {
        $this->descricao = $descricao;
    }

    function setIdData($idData) {
        $this->idData = $idData;
    }



    
}
